auth/presentation/src/main/java/com/nibalk/tasky/auth/presentation/login/LoginViewModel.kt

```diff
@@ -56,8 +65,25 @@ class LoginViewModel(
 
     private fun login() {
         viewModelScope.launch {
-            eventChannel.send(LoginEvent.LoginSuccess)
-            // TODO: Make the API call here
+            state = state.copy(isLogin = true)
```

Comment:
I think the correct wording would be `isLoggingIn` if you want to refer to whether you're loading or not, right?